### PR TITLE
Check jwt key on startup

### LINKED ISSUE #125: Ensure the JWT signing key exists on start up.
Currently it is not checked that the key exists until it is being used.

### PR DESCRIPTION
**Pull request by aaperis**
This PR adds checking that the private jwt key which is provided in the configuration corresponds to an existing file. The check is performed as early as possible at the `config` module and the app will therefore exit with error if it cannot find the key file
upon startup.

Closes #125

### CODE REVIEW

File: config.go [hunk 1]
```diff
@@ -163,5 +165,9 @@ func (c *Config) readConfig() error {
 		}
 	}
 
+	if _, err := os.Stat(c.JwtPrivateKey); errors.Is(err, os.ErrNotExist) {
+		return fmt.Errorf("Error when reading from private key file, reason: '%s'", err)
```
Comment (jbygdell) [inline config.go:169]: Maybe reword the error message since the file is clearly missing.

**Review by norling — APPROVED** [review]

**Review by dbampalikis — APPROVED** [review]
